client/examples/http_client/response.rs

```diff
@@ -96,9 +100,11 @@ impl fmt::Display for Transfer {
     }
 }
 
+#[serde_with::serde_as]
 #[derive(Serialize)]
 pub(crate) struct FindNode {
     pub node: Node,
+    #[serde_as(as = "serde_with::DurationMilliSeconds<u64>")]
     pub duration: Duration,
```

Comment:
Why not just:
```rust
pub duration_ms: u128,
```
?

Reply:
(I received comment notification with some delay)